Soccar [UI]/src/nl/soccar/ui/fx/controller/LoginFXMLController.java

```diff
@@ -1,17 +1,24 @@
 package nl.soccar.ui.fx.controller;
 
 import java.net.URL;
+import java.rmi.ConnectException;
+import java.rmi.RemoteException;
 import java.util.ResourceBundle;
+import java.util.logging.Level;
+import java.util.logging.Logger;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.logging.Logger'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)